### ISSUE TITLE
There are multiple templates named 'override-atPwdForm'...

### ISSUE THREAD

**Issue by Kamin12** [meteor-useraccounts/core#710]
So I have an error in the browser that says "Uncaught Error: There are multiple templates named 'override-atPwdForm'. Each template needs a unique name."

....

Except I have deleted all traces of 'override-atPwdForm'....

It seems that accounts-core has a problem computing and holding on to old code, that no longer exists. Why is this template still being computed if I deleted all traces of it? Is there any other reason why this error is still occuring?

**Comment by MichaelJCole** [comment]
I don't think this has anything to do with this package.  This is a message from meteor.  There is a bug in your code.
